frame/curve-amm/src/lib.rs

```diff
@@ -824,6 +769,73 @@ pub mod pallet {
 	}
 
 	impl<T: Config> Pallet<T> {
+		pub fn create_pool(
+			who: &T::AccountId,
+			assets: Vec<T::AssetId>,
+			amplification_coefficient: FixedU128,
+			fee: Permill,
+			admin_fee: Permill,
+		) -> Result<T::PoolId, DispatchError> {
+			// Assets related checks
+			ensure!(assets.len() > 1, Error::<T>::NotEnoughAssets);
```

Comment:
Vec is unbounded. so if we will be success and have as many assets as FTX, it can easy be 1000 of assets. what is plan to have this method secured in terms of weight? seems as count of assets grows, grows price of this method with no limit.

Reply:
I will remove Vec in next PR.